source/_data/themes.yml

```diff
@@ -3085,3 +3085,14 @@
     - elegant
     - clear
     - responsive
+- name: WordKit
+  description: A Hexo Theme Based on UIKit Front-end Framework.
+  link: https://github.com/ThusFun/hexo-theme-wordkit
+  preview: https://hexo-wordkit.thus.fun/
+  tags:
+    - simple
+    - clear
+    - responsive
+    - UIKit
+    - 中文
+    - Chinese
```

Comment:
Would you mind adding a new line at the end of fileq?

Reply:
Do you mean let me add a blank line?